_posts/2024-04-15-gsi-tech-blog.md

```diff
@@ -0,0 +1,75 @@
+---
+# List of one or more author names that map to the value of short_name in site.community_members. See the content in the _community_members collection for reference.
+authors: 
+    - nateboot
+
+# Content categorization. One or more categories can be specified. 
+category:
+  - community
+
+# Publish date in the format of four digit year, two digit month, two digit day, hour, minute, second, and timezone offset; e.g., 2021-04-12 01:01:01 -0700
+date: 2024-04-15 01:01:01 -0700
+
```

Comment:
Please add meta: 

meta_keywords: ML model, vector store, GSI Technology, OpenSearch AI and ML

meta_description: GSI Technology uses OpenSearch as a vector store to solve advanced real world problems in the realms of AI, Aerospace, Defense and even DNA mapping.